mycore-base/src/main/java/org/mycore/datamodel/metadata/MCRDefaultMetaDerivateLinkIDFactory.java

```diff
@@ -0,0 +1,49 @@
+package org.mycore.datamodel.metadata;
+
+import java.util.ArrayList;
+
+import org.jdom2.Content;
+import org.jdom2.Element;
+import org.mycore.common.MCRConstants;
+
+public class MCRDefaultMetaDerivateLinkIDFactory extends MCRMetaDerivateLinkIDFactory{
+
+    @Override
+    public MCRMetaDerivateLinkID getDerivateLink(MCRDerivate der) {
```

Comment:
we should think about naming here
 - MCRMetaStructureDerObject
 - MCRMetaDerObject